lms/exceptions.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+"""Exceptions raised by lti launch code."""
+
+from __future__ import unicode_literals
+
+from pyramid.i18n import TranslationString as _  # noqa: N813
+
+
+class LtiLaunchError(Exception):
```

Comment:
Pyramid actually comes with a [set of exception classes representing HTTP statuses](https://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html), and the status that we want for LTI launch param errors is `400` (`HTTPBadRequest`). I wonder if `LTILaunchError` should be a subclass of Pyramid's `HTTPBadRequest`?